src/stories/beta/Form.js

```diff
@@ -36,16 +36,24 @@ const basicPokedexSchema = {
       type: 'string'
     },
     pokedex_number: {
-      title: 'National Pokedex Number',
+      title: 'National Pokèdex Number',
       type: 'number'
     },
     caught: {
       type: 'boolean'
     },
     first_seen: {
       title: 'First seen',
+      description: 'The first time you saw this pokèmon',
       type: 'string',
       format: 'date-time'
+    },
+    Tags: {
```

Comment:
The `Tags` label is rendered differently when compared with the rest of the titles (eg `Tags` is bold while `FIRST SEEN` isn't and shows up in uppercase).
How about re-using the `FieldTemplate.Label`?